scripts/tier-picker-smoke.cjs

```
@@ -0,0 +1,104 @@
+    cwd: ROOT,
+    env: { ...process.env, PORT: String(PORT) },
+    stdio: 'ignore',
+  });
+
+  try {
+    await waitForServer(BASE);
+    const browser = await chromium.launch({ args: ['--no-sandbox'] });
+    const page = await browser.newPage();
+    await page.goto(`${BASE}/app/`, { waitUntil: 'load' });
+    await page.waitForFunction(
+      () => window._vipApp && document.querySelector('[data-hero-tier]'),
```

ESLint

[error] 'window' is not defined.

(no-undef)